mypyc/lib-rt/CPy.h

```diff
@@ -936,6 +936,24 @@ static PyObject *CPyIter_Send(PyObject *iter, PyObject *val)
     }
 }
 
+static PyObject *CPy_GetCoro(PyObject *obj)
+{
+    // If the type has an __await__ method, call it,
+    // otherwise, fallback to calling __iter__.
+    PyAsyncMethods* async_struct = obj->ob_type->tp_as_async;
+    if (async_struct != NULL) {
+        if (async_struct->am_await != NULL) {
+            return (async_struct->am_await)(obj);
+        }
+        else {
+            return PyObject_GetIter(obj);
+        }
+    }
+    else {
```

Comment:
Putting the `else {` on a separate line *is* what PEP 7 (Python's "Style Guide for C Code") calls for.

But I really hate it, so all of mypyc's C code cuddles elses like `} else {`.

Reply:
This is done